### DISCUSSION TITLE
Idea(use-case): Internet Archive & Block Protocol?

### DISCUSSION THREAD

**Discussion by ghost** [blockprotocol/blockprotocol#592] [Q&A] [answered]
Hi. 

I would like to know if you have ever thought about componentizing these 732 billion pages of the Internet Archive project. I was thinking of creating an algorithm that takes the css of all Internet Archive pages and generates the components in Block Protocol. 

The reason for this request would be to analyze with you whether this proposal is viable or not. Another reason, imagine having 732 billion web pages component types?

*In theory this would be possible with python with selenium and this extension called css used.*

**Comment by vilkinsons** [comment, marked as answer]
Hi @codehangen, and welcome! I'm not sure I see the utility in this approach versus just having one 'Internet Archive' block which could accept parameters enabling it to load any of the pages in its index. If you've a specific use-case in mind it would be interesting to explore.
